src/shared/store/store.ts

```diff
@@ -0,0 +1,17 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+import { userSlice } from './user/userSlice';
+
+export const store = configureStore({
+  reducer: {
+    user: userSlice.reducer,
+  },
+});
```

Comment:
Предлагаю добавить, вроде это позводяет следить за стором в дев тулс, нагуглил, но опыта не имею.
```suggestion
  },
  devTools: process.env.NODE_ENV !== 'production',
});
```